internal/scanner/http_client.go

```diff
@@ -36,6 +37,8 @@ func NewHTTPClient(cfg *config.Config) *HTTPClient {
 		}
 	}
 
+	jar, _ := cookiejar.New(nil)
```

Comment:
Why errors are ignored?

Reply:
Here is the implementation of `New`:
```golang
func New(o *Options) (*Jar, error) {
	jar := &Jar{
		entries: make(map[string]map[string]entry),
	}
	if o != nil {
		jar.psList = o.PublicSuffixList
	}
	return jar, nil
}
```
error is always `nil`, especially in the case of empty initialization.

But I fixed it in case of future lib changes, thanks. 2c99196